docs/how-to-create-tests.md

```diff
@@ -84,12 +80,12 @@ The client class contains an endpoint for the URL of the resource. This class cr
 Perform the following steps to create a simple (zero dependents) client:
 
 1. Create the file for the resource client:
-    * ~Suite-3-Performance-Testing\edfi_performance\api\client\course.py
+    * ~Suite-3-Performance-Testing\src\edfi-performance-test\edfi_performance_test\api\client\course.py
     * Replace 'course' with the name of your new resource
 2. Add the following import statements to the top of the file (Your import for the factory may look different):
 
    ```python
-   from edfi_performance.api.client import EdFiAPIClient
+  from edfi_performance_test.api.client.ed_fi_api_client import EdFiAPIClient
```

Comment:
The preview for this section is not loading correctly. See it here: https://github.com/Ed-Fi-Exchange-OSS/Suite-3-Performance-Testing/blob/PERF-277/docs/how-to-create-tests.md#2-client

After testing locally it seems related to the tabs in the area. This loads on the VS Code Preview
![image](https://user-images.githubusercontent.com/5459528/192345828-87dd30d1-e0c5-40ea-add4-1726f2b8dde1.png)